prow/plugins/milestone/milestone.go

```diff
@@ -165,3 +165,11 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent, r
 
 	return nil
 }
+
+func determineMaintainers(gc githubClient, milestone plugins.Milestone, org string) ([]github.TeamMember, error) {
```

Comment:
Can we in the config loading once during startup log a warning if `.MaintainersID`is set  that tells ppl to use `MaintainersTeam` instead?

Reply:
Thats a good idea. It also prompted me to remove test-infra's usages of the `maintainers_id`.